include/logging/log_core.h

```diff
@@ -14,6 +14,40 @@
 extern "C" {
 #endif
 
+/* MOJ KOD */
+
+#include <zephyr.h>
+#include <device.h>
+#include <gpio.h>
+#include <hal/nrf_gpio.h>
+
+#if (CONFIG_LOG_FRONTEND_PORT_NUMBER == 0)
```

Comment:
this should go to log_frontend_nrf_gpio.c file

Reply:
you should create log_frontend_nrf_gpio.c (for now it can be in zephyr/subsys/logging)